AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -18,6 +18,22 @@ void ASPerformBlockOnMainThread(void (^block)());
 void ASPerformBlockOnBackgroundThread(void (^block)()); // DISPATCH_QUEUE_PRIORITY_DEFAULT
 ASDISPLAYNODE_EXTERN_C_END
 
+/**
+ * Bitmask to indicate what performance measurements the cell should record.
+ */
+typedef NS_OPTIONS(NSUInteger, ASDisplayNodePerformanceMeasurementOptions) {
+  ASDisplayNodePerformanceMeasurementOptionLayoutSpec = 1 << 0,
+  ASDisplayNodePerformanceMeasurementOptionLayoutGeneration = 1 << 1
```

Comment:
@shamanskyh these are so awesome to have!  I might suggest the second one be LayoutMeasurement or LayoutComputation.  Not important enough for a follow-up diff now though.